pkg/api/v1/zz_generated.defaults.go

```diff
@@ -145,9 +145,6 @@ func SetObjectDefaults_PersistentVolume(in *v1.PersistentVolume) {
 	if in.Spec.PersistentVolumeSource.AzureDisk != nil {
 		SetDefaults_AzureDiskVolumeSource(in.Spec.PersistentVolumeSource.AzureDisk)
 	}
-	if in.Spec.PersistentVolumeSource.ScaleIO != nil {
-		SetDefaults_ScaleIOVolumeSource(in.Spec.PersistentVolumeSource.ScaleIO)
```

Comment:
need to copy `SetDefaults_ScaleIOVolumeSource` to `SetDefaults_ScaleIOPersistentVolumeSource` (changing the types), which will regenerate this to call a defaulting function

Reply:
which also means copying the registered fuzzing function in `pkg/api/fuzzer/fuzzer.go` for `func(sio *api.ScaleIOVolumeSource, c fuzz.Continue)` for ScaleIOPersistentVolumeSource, and registering the new type in `pkg/api/testing/defaulting_test.go#TestDefaulting`

Reply:
ok will do.